### PR TITLE
Changed fallback to application/problem+json

### PR DESCRIPTION
**Pull request by whiskeysierra**
See #124

### CODE REVIEW

File: src/test/java/org/zalando/problem/spring/web/advice/http/NotAcceptableAdviceTraitTest.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ void notAcceptable() throws Exception {
     @Test
     void notAcceptableNoProblem() throws Exception {
         mvc().perform(request(GET, "http://localhost/api/handler-ok")
-                .accept("application/atom+xml"))
+                .accept("image/png"))
                 .andExpect(status().isNotAcceptable())
                 .andExpect(header().doesNotExist("Content-Type"));
```
Comment (yvandecn) [inline src/test/java/org/zalando/problem/spring/web/advice/http/NotAcceptableAdviceTraitTest.java]: Shouldn't the `Content-Type` be `MediaTypes.PROBLEM` ?

Comment (whiskeysierra) [reply src/test/java/org/zalando/problem/spring/web/advice/http/NotAcceptableAdviceTraitTest.java]: Yes. I just haven't figured out a way yet to force Spring to do the right thing there. The content negotiation is done up front and my code never runs...

Comment (yvandecn) [reply src/test/java/org/zalando/problem/spring/web/advice/http/NotAcceptableAdviceTraitTest.java]: You can ignore the Accept header in the Spring Configuration using something like this:
```
public void configureContentNegotiation(ContentNegotiationConfigurer configurer) {

	configurer.favorPathExtension(false);
	configurer.ignoreAcceptHeader(true);
	configurer.defaultContentType(MediaType.APPLICATION_JSON);

}
```
You could try adding this to your test configuration.

**Comment by whiskeysierra** [comment]
👍

**Pull request merged**